docs/zh/start/trace/zipkin.md

```diff
@@ -0,0 +1,93 @@
+# ZipKin trace 接入
+
+## 配置
+
+示例：configs/config_trace_zipkin.json
+
+```json
+{
+  "tracing": {
+    "enable": true,
+    "driver": "Zipkin",
+    "config": {
+      "config": {
+        "config": {
+          "service_name": "layotto",
+          "reporter_endpoint": "http://127.0.0.1:9411/api/v2/spans",
+          "recorder_host_post": "127.0.0.1:9000"
+        }
+      }
+    }
+  }
+}
+
+```
+| 字段   | 必填  | 说明                                           |
+|------|-----|----------------------------------------------|
+| service_name | Y   | 服务名称，默认为layotto                              |
+| reporter_endpoint | Y   | 链路日志输出，默认为http://127.0.0.1:9411/api/v2/spans |
+| recorder_host_post     | Y   | 当前服务端口号，默认为127.0.0.1:9000                                  |
+
+注意：目前只支持Http方式的Reporter。
+
+## 运行ZipKin
+
+```shell
+cd ${project_path}/diagnostics/zipkin
+
+docker-compose -f zipkin-docker-compose.yaml up -d
+```
+
+## 运行layotto
+
+可以按照如下方式启动一个layotto的server：
```

Comment:
Hi, could u also add a section showing how to run layotto with docker ? That's helpful for windows users.
I submitted a PR #660 and you can take that as an example.
The trick here is using the `--network` flag to connect different docker containers.  
![image](https://user-images.githubusercontent.com/26001097/174709556-eafd7e35-8ec6-41ae-aae9-31af03a24ae6.png)

For your case , you can use `--network=zipkin_default` to connect layotto container and zipkin containers